trace_context/HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -31,6 +31,22 @@ interfere with identification of the tracing system responsible for an entry.
 - The typical name will be a single word in latin and the value will be a
 copy of the `traceparent` format or an opaque string.
 
+## Ordering of keys in `tracestate`
+
+Specification calls for ordering of values in tracestate. This requirements allows better interoperability between tracing vendors.
```

Comment:
I think this md file uses different line size based on whoever updates it. Probably makes sense to format the entire file using some rules. Maybe in a different PR. But at least in this PR use consistent line sizes please.

Reply:
I noticed this as well. I wonder if there is a setting common between vim and vscode that will control it. I'll configure this for vscode.